torch/_dynamo/logging.py

```diff
@@ -78,8 +83,24 @@ def init_logging(log_level, log_file_name=None):
 
 _step_counter = itertools.count(1)
 
+# Update num_steps if more phases are added: Dynamo, AOT, Backend
+# This is very inductor centric
+# _inductor.utils.has_triton() gives a circular import error here
+
+if not disable_progress:
+    try:
+        import triton  # noqa: F401
```

Comment:
Should you also check for device capability? (`get_device_capability() >= (7, 0)` in `has_triton`)?

Reply:
TBH I don't like either, I'd prefer to check if dynamo is using the inductor backend but again that makes dependencies circular